appeals/web/src/server/appeals/appeal-details/withdrawal/withdrawal.types.d.ts

```diff
@@ -0,0 +1,4 @@
+export interface WithdrawalRequest {
+	withdrawalRequestDate?: Date | null;
+	redactionStatus?: string | null;
```

Comment:
Just a thought, I think this is fine, but this could potentially be typed more strictly to only be one of the values in `REDACTION_STATUS`. I think this should work, but feel free to ignore me if not :D 

`redactionStatus?: typeof REDACTION_STATUS[keyof typeof REDACTION_STATUS] | null;`

Assuming you import `import { REDACTION_STATUS } from '@pins/appeals/constants/documents.js';`

Reply:
I wish there was a less ugly way to do this in TS...

Reply:
Not done. Explained in the chat